.github/workflows/pythonapp.yml

```diff
@@ -2,20 +2,30 @@ name: testing cihub
 
 on:
   push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+    type: [ "opened", "reopened", "synchronize" ]
   schedule:
     - cron: '0 5 * * MON-FRI'
 
 jobs:
   build:
-
+    name: run python ${{ matrix.python-version }} tests
     runs-on: ubuntu-latest
```

Comment:
Also mal so grundsätzlich und nicht auf den Kommentar: Will man das tatsächlich? in 6 Monaten ist das schon wieder was anderes.